### DISCUSSION TITLE
Outputting the last time step

### DISCUSSION THREAD

**Discussion by rnpratoori** [idaholab/moose#28449] [Q&A Modules: General] [answered]
Hello team,

I am using `time_step_interval` option to create output files at an interval. With this, I noticed that the last time step does not always get written because of the above option. How can I get the last time step irrespective of what `time_step_interval` I use?

**Comment by GiudGiud** [comment, marked as answer]
Hello

You can add the execute_on  FINAL option to the output.
So probably set the parameter as execute_on = 'TIMESTEP_END FINAL' 

Guillaume
